src/main/java/cc/factorie/util/tpt/ByteArray.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * This file is part of "FACTORIE" (Factor graphs, Imperative, Extensible)
+ * http://factorie.cs.umass.edu, http://github.com/factorie
+ * Licensed to the University of Massachusetts Amherst (UMass) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * UMass licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Copyright C 2016, Oracle and/or its affiliates. All rights reserved.
```

Comment:
We need to add a line in LICENSE.txt like we did with AhoCorasick

Reply:
I don't see any Oracle copyright in LICENSE.txt.  So, I'm not sure what is required here.